src/test/java/com/microsoft/sqlserver/jdbc/AlwaysEncrypted/RegressionAlwaysEncryptedTest.java

```diff
@@ -228,9 +230,43 @@ private void verifyNumericTable(Connection connection, boolean isNull) throws SQ
         }
     }
 
+    @ParameterizedTest
+    @MethodSource("enclaveParams")
+    public void testStringColumnEncryptWithVaryLength(String serverName, String url, String protocol) throws Exception {
+        setAEConnectionString(serverName, url, protocol);
+        try (Connection connection = PrepUtil
+                .getConnection(AETestConnectionString + ";columnEncryptionSetting=enabled;", AEInfo);
+                Statement stmt = connection.createStatement()) {
+            dropTables(stmt);
+
+            createTable(VARY_STRING_TABLE_AE, cekJks, stringTable);
+
+            verifyStringTable(connection);
```

Comment:
Make sure to clean up even if the test errors. Bonus points if you improve the two existing tests above to clean up on error, too.
```suggestion
            try {
                verifyStringTable(connection);
            } finally {
                dropTables(stmt);
            }
```